connect/mirror/src/main/java/org/apache/kafka/connect/mirror/ConfigPropertyFilter.java

```diff
@@ -27,6 +27,13 @@ public interface ConfigPropertyFilter extends Configurable, AutoCloseable {
 
     boolean shouldReplicateConfigProperty(String prop);
```

Comment:
Can we add a note to this method that it does not handle replication of default properties, and that that logic is instead controlled by `shouldReplicateSourceDefault`?

Reply:
(Depends on conversation on a different thread)